dts/bindings/misc/intel,timeaware-gpio.yaml

```diff
@@ -20,3 +20,10 @@ properties:
     type: int
     required: true
     description: Total number of available pins
+
+  artv-ctrl:
+    type: boolean
+    description: Some platforms have ARTV_CTRL which is required for
+                 reading the ART timestamp. This property is used to
+                 indicate if the support of ARTV_CTRL is available
+                 or not.
```

Comment:
nit but I don't think I've ever seen any binding us this type of formatting/text alignment

Reply:
Updated, please review.